package.json

```diff
@@ -7,32 +7,60 @@
   },
   "private": true,
   "dependencies": {
+    "@angular/animations": "~19.1.0",
+    "@angular/common": "~19.1.0",
+    "@angular/compiler": "~19.1.0",
+    "@angular/core": "~19.1.0",
+    "@angular/fire": "^19.0.0",
+    "@angular/forms": "~19.1.0",
+    "@angular/platform-browser": "~19.1.0",
+    "@angular/platform-browser-dynamic": "~19.1.0",
+    "@angular/pwa": "^19.1.6",
+    "@angular/router": "~19.1.0",
+    "@angular/service-worker": "^19.0.0",
+    "@tailwindcss/postcss": "^4.0.6",
     "axios": "^1.6.0",
     "class-transformer": "^0.5.1",
     "firebase": "^11.2.0",
     "firebase-admin": "^13.0.2",
     "firebase-functions": "^6.3.1",
+    "postcss": "^8.5.1",
     "reflect-metadata": "^0.2.2",
-    "serializr": "^3.0.3"
+    "rxjs": "~7.8.0",
+    "serializr": "^3.0.3",
+    "tailwindcss": "^4.0.6",
+    "zone.js": "~0.15.0"
   },
   "devDependencies": {
+    "@angular-devkit/build-angular": "~19.1.0",
+    "@angular-devkit/core": "~19.1.0",
+    "@angular-devkit/schematics": "~19.1.0",
+    "@angular/cli": "~19.1.0",
+    "@angular/compiler-cli": "~19.1.0",
+    "@angular/language-service": "~19.1.0",
     "@eslint/js": "^9.18.0",
+    "@nx/angular": "^20.4.1",
     "@nx/esbuild": "20.4.0",
     "@nx/eslint": "20.4.0",
     "@nx/eslint-plugin": "20.4.0",
     "@nx/jest": "20.4.0",
-    "@nx/js": "20.4.0",
+    "@nx/js": "20.4.1",
```

Comment:
This might cause issues (nx schemas/packages out of sync with the others), but no need to change. I will update the other nx libs soon.